qemu/tests/cfg/device_assignment.cfg

```diff
@@ -1,13 +1,26 @@
 - device_assignment:
     only pf_assignable
     virt_test_type = qemu
+    pre_command = "modprobe vfio-pci"
```

Comment:
Yes, but we don't need to set the flag for device_assignment. sr_iov_setup() is used to set vf for pf.  No vf is needed in this case.